planner/core/physical_plan_test.go

```diff
@@ -864,7 +864,7 @@ func (s *testPlanSuite) TestDAGPlanBuilderAgg(c *C) {
 		// Test agg + table.
 		{
 			sql:  "select sum(a), avg(b + c) from t group by d",
-			best: "TableReader(Table(t)->HashAgg)->HashAgg",
+			best: "TableReader(Table(t))->Projection->HashAgg",
```

Comment:
Why this one changed?

Reply:
I felt confused too. So I'm investigating the reason now.

Reply:
I guess it should be caused by the fact that the number of `AggFuncs` increases, so the cost of Aggregation is changed, and planner decides not to push this Agg down to coprocessor.

Reply:
According to https://github.com/pingcap/tidb/pull/11926#issuecomment-526050107, seems we should modify the cost model to decrease the impact of number of aggregate functions to make this aggregation can be pushed to tikv?

Reply:
We can temporarily not to modify the cost model. That's another task of cost model improvement with the consideration of aggregation type.

Reply:
OK, How about filing an issue about this improvement?

Reply:
Okay.